index.html

```diff
@@ -210,81 +210,115 @@ <h2>Multi-Post
+                </div>
             </div>
-            <div class="about-tech">
-                <ul class="drop">
-                    <li class="drop-link border-none">
-                        <span>
-                            Languages
-                            <i class="fa-solid fa-angle-down"></i>
-                        </span>
-                        <ul class="drop-item none-style">
-                            <li class="drop-down">
-                                <i class="fa-brands fa-html5 bg-html-color" ></i> HTML
-                            </li>
-                            <li class="drop-down">
-                                <i class="fa-brands fa-css3-alt bg-css-color"></i>
-                                CSS
-                            </li>
-                            <li class="drop-down">
-                                <i class="fa-brands fa-js bg-js-color"></i>
-                                JavaScript
-                            </li>
-                        </ul>
-                    </li>
-                    <li class="drop-link">
-                        <span>
-                            Framworks
-                            <i class="fa-solid fa-angle-right"></i>
-                        </span>
-                    </li>
-                    <li class="drop-link">
-                        <span>
-                            Skills
-                            <i class="fa-solid fa-angle-right"></i>
-                        </span>
-                    </li>
-                </ul>
+        </section>
+        <section class="contact">
+            <div class="contact-form-container flex-box">
+                <div class="contact-body flex-box">
+                    <div class="contact-title">
+                        <h2>Contact me</h2>
+                    </div>
+                    <div class="contact-paragraph">
+                        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+                            I’d love to help with it!</p>
+                    </div>
+                </div>
+                <div class="container-form">
+                    <div class="contact-form ">
+                        <form action="https://formspree.io/f/mgebnweb"
+                        method="POST">
+                            <div class="input-field">
+                                <input type="text" maxlength="30" placeholder="Lorem Ipsum">
+                            </div>
+                            <div class="input-field">
+                                <input type="email" placeholder="loremipsum@gmail.com">
+                            </div>
+                            <div class="input-field message">
+                                <textarea name="message" maxlength="500" cols="30" rows="10" placeholder="Write your message here..."></textarea>
+                            </div>
```

Comment:
- [ ] All input fields have the `required` attribute to activate the validation on submit